snippets/3.4/windows.html.in

```diff
@@ -2,7 +2,6 @@
   <p>
     The following packages are for Windows 7, Windows 10.1, Windows 2008 R2 and higher.
```

Comment:
Windows 10.1 is unoffical naming. Also, it's 1809 build while I'm on 1803 and it works fine. Also, Windows 7 and Windows 2008 R2 could be specialized.

I propose the following: `Windows 7 Service Pack 1, Windows 10 (1803 and higher), Windows Server 2008 R2 and higher`.

Reply:
like that.

Reply:
@KVS85 I like your proposal.
So Windows 7 without Service Packs doesn't work? Do you happen to know the reason?

Reply:
Windows 7 without SP1 is not supported by Microsoft: https://support.microsoft.com/en-us/help/13853/windows-lifecycle-fact-sheet. I suppose, we won't too.